sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/DirectoriesCreateHeaders.java

```diff
@@ -89,9 +112,13 @@ public final class DirectoriesCreateHeaders {
 
     private static final HttpHeaderName X_MS_FILE_CREATION_TIME = HttpHeaderName.fromString("x-ms-file-creation-time");
 
+    private static final HttpHeaderName X_MS_FILE_FILE_TYPE = HttpHeaderName.fromString("x-ms-file-file-type");
```

Comment:
Is this header name correct, or is there a typo here?

x-ms-file-file-type

Reply:
yep, its correct! this is also generated code, meaning these values are created through the swagger. you can tell if I file contains generated code by viewing the first couple of lines of the file, it will say `// Code generated by Microsoft (R) AutoRest Code Generator.` on line 3 if it is generated

Reply:
Sure, got it! I might have missed that this was part of the AutoRest-generated code. You can ignore this.